### PR TITLE
Update antlr4ts version and add antlr build script

### LINKED ISSUE #2206: antlr4ts scripts are missing from adaptive-expressions/package.json
We are missing script support for the antlr4 generation.  This makes it hard to upgrade the package and means there is knowledge outside the repo.

The package.json file should have:
1) A script for generating the antlr4 files like: "antlr4ts": "antlr4ts -visitor src/CommonRegex.g4 -o generated",
2) A dev dependency like: "antlr4ts-cli": "^0.5.0-alpha.3",  (We should upgrade)

[bug]

### PR DESCRIPTION
**Pull request by Danieladu**
Fixes #2206

1. update `antlr4ts` and `antlr4ts-cli` from "0.5.0-alpha.1" to "0.5.0-alpha.3" (which update the Antlr core library from 4.6 to 4.7.3)
2.Add `antlr-build-expression` and `antlr-build-commonregex` in expression, 
and add `antlr-build` script in language-generation.

### CODE REVIEW

File: libraries/adaptive-expressions/src/parser/generated/ExpressionAntlrParserListener.ts [hunk 1]
```diff
@@ -85,17 +98,17 @@ export interface ExpressionAntlrParserListener extends ParseTreeListener {
 	exitStringAtom?: (ctx: StringAtomContext) => void;
 
 	/**
-	 * Enter a parse tree produced by the `indexAccessExp`
+	 * Enter a parse tree produced by the `idAtom`
 	 * labeled alternative in `ExpressionAntlrParser.primaryExpression`.
 	 * @param ctx the parse tree
 	 */
-	enterIndexAccessExp?: (ctx: IndexAccessExpContext) => void;
+	enterIdAtom?: (ctx: IdAtomContext) => void;
```
Comment (stevengum) [inline libraries/adaptive-expressions/src/parser/generated/ExpressionAntlrParserListener.ts:105]: This looks like a breaking change as [`IdAtomContext`](https://github.com/microsoft/botbuilder-js/blob/208f0fc7a4c535f7861ab95e9683a70b6d118545/libraries/adaptive-expressions/src/parser/generated/ExpressionAntlrParser.ts#L1322) does not have the same signature of [`IndexAccessExpContext`](https://github.com/microsoft/botbuilder-js/blob/8c15615dd5cf6dc6817a0771cb7ea3bf85e07455/libraries/adaptive-expressions/src/parser/generated/ExpressionAntlrParser.ts#L1295).

Comment (Danieladu) [reply libraries/adaptive-expressions/src/parser/generated/ExpressionAntlrParserListener.ts:105]: there is no breaking change on parser, it is just the misconception caused by the dislocation of the code diff.

File: libraries/adaptive-expressions/package.json [hunk 2]
```diff
@@ -41,14 +41,17 @@
     "mocha": "^5.2.0",
     "nyc": "^15.0.0",
     "ts-node": "^4.1.0",
-    "typescript": "3.5.3"
+    "typescript": "3.5.3",
+    "antlr4ts-cli": "^0.5.0-alpha.3"
   },
   "scripts": {
-    "build": "tsc",
+    "build": "npm run antlr-build-expression && npm run antlr-build-commonregex && tsc",
```
[the remark this answers is not in the document]
Comment (Danieladu) [reply libraries/adaptive-expressions/package.json:48]: add "npm run build" into "test" script

File: libraries/adaptive-expressions/package.json [hunk 3]
```diff
@@ -24,7 +24,7 @@
     "@types/lru-cache": "^5.1.0",
     "@types/moment-timezone": "^0.5.12",
     "@types/xmldom": "^0.1.29",
-    "antlr4ts": "0.5.0-alpha.1",
+    "antlr4ts": "^0.5.0-alpha.3",
```
[the remark this answers is not in the document]
Comment (Danieladu) [reply libraries/adaptive-expressions/package.json]: Update the antlr4ts version and pin the version.

**Pull request merged**